ebpf-for-windows-demo.sln

```diff
@@ -9,37 +9,129 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "daemon", "cilium\load_balan
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "bpf_xdp", "cilium\load_balancer\bpf\bpf_xdp\bpf_xdp.vcxproj", "{742B767C-02C3-4D5A-81A9-232D57262388}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "tests", "tests", "{9171E26A-58B7-4DAB-B00B-C9F0F667D6D5}"
+EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "cilium_test", "tests\cilium_tests\cilium_test.vcxproj", "{12161211-5463-4AA9-9B6E-552552B475DC}"
+EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Catch2WithMain", "external\Catch2\build\src\Catch2WithMain.vcxproj", "{8BD3552A-2CFB-4A59-AB15-2031B97ADA1E}"

```

Comment:
if you are using catch as  a submodule and linking as a lib, then you dont need nuget package.config for catch.

Reply:
Updated issue #8 to fix this issue. Currently it is not possible to fix this as demo repo is including catch_wrapper.hpp from ebpf-for-windows submodule, and the submodule is pointing to an older commit.